packages/@aws-cdk/aws-stepfunctions-tasks/test/ecs-tasks.test.ts

```diff
@@ -162,7 +161,7 @@ test('Running an EC2 Task with bridge network', () => {
       {
         containerName: 'TheContainer',
         environment: [
-          {name: 'SOME_KEY', value: JsonPath.stringFromPath('$.SomeKey')}
+          {name: 'SOME_KEY', value: sfn.DataField.fromStringAt('$.SomeKey')}
```

Comment:
I don't think that the `from` prefix is required. This reads better: `sfn.DataField.stringAt('$.SomeKey')`, no? 

I also think that perhaps `sfn.Field` for data fields is good and `sfn.Context`?

Reply:
Agreed on the method names, but not sure on the class names. `Field` feels vague to me, and `Context` is inaccurate.